system-addon/test/unit/lib/Store.test.js

```diff
@@ -38,55 +49,129 @@ describe("Store", () => {
     store.dispatch({type: "FOO"});
     assert.calledOnce(store._mm.middleware);
   });
-  describe("#init", () => {
-    it("should dispatch an init action", () => {
-      sandbox.spy(store, "dispatch");
+  describe("#initFeed", () => {
+    it("should add an instance of the feed to .feeds", () => {
+      class Foo {}
+      store._feedConstructors = {foo: () => new Foo()};
 
-      store.init();
+      store.initFeed("foo");
 
-      assert.calledWith(store.dispatch, {type: at.INIT});
+      assert.isTrue(store.feeds.has("foo"), "foo is set");
+      assert.instanceOf(store.feeds.get("foo"), Foo);
     });
-    it("should dispatch an init action", () => {
-      sandbox.spy(store, "dispatch");
+    it("should add a .store property to the feed", () => {
+      class Foo {}
+      store._feedConstructors = {foo: () => new Foo()};
+      store.initFeed("foo");
 
-      store.init([]);
-
-      assert.calledWith(store.dispatch, {type: at.INIT});
+      assert.property(store.feeds.get("foo"), "store");
+      assert.equal(store.feeds.get("foo").store, store);
```

Comment:
I'm not sure.  I've just sort of gotten used to current reality that there are lots of good, specific assertions, so typically when I notice a .equal, I immediately wonder if there isn't something a bit clearer... :-)